### PR TITLE
Designed the MainDashboard

### PR DESCRIPTION
**Pull request by plakshiyadav12**
Screenshot (15)

### CODE REVIEW

**Review by akanoao — COMMENTED** [review]
all good

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@mui/types": "^7.2.19",
```
Comment (geek-room-community) [inline package.json:13]: please try using `"@mui/material": "latest",` here

Comment (plakshiyadav12) [reply package.json:14]: ok

Comment (geek-room-community) [reply package.json:14]: Why didn't these changes reflect in package-lock.json?

**Review by geek-room-community — APPROVED** [review]

**Pull request merged**